ark/utils/notebooks_test.py

```diff
@@ -31,11 +31,6 @@ def _exec_notebook(nb_filename, base_folder):
         subprocess.check_call(args)
 
 
-# test runs with default inputs
-def test_segment_image_data():
-    _exec_notebook('1_Segment_Image_Data.ipynb', 'templates')
```

Comment:
How come this isn't needed anymore? Does `test_segment_image_data_folder` cover the necessary cases?

Reply:
It would run the whole notebook with the entire dataset. Removing it cuts time, however as a consequence that's why the [Coverage dropped 1.3%](https://coveralls.io/builds/53260253/source?filename=ark%2Futils%2Fdeepcell_service_utils.py#L188).  `run_deepcell_direct` is only when the notebook is ran.